starter/angular.json

```diff
@@ -19,8 +19,8 @@
             "assets": [
               {
                 "glob": "mdi.svg",
-                "input": "node_modules/@mdi/angular-material",
-                "output": "./assets/icons"
+                "input": "../node_modules/@mdi/angular-material",
```

Comment:
Now @mdi/angular-material dependency is part of stark-ui, I think we should move this into angular.json file of stark-ui

Reply:
Mmm in fact I didn't add it to the angular.json on purpose, why? because if the app doesn't want to add the @mdi icons, they will be included anyways in the /assets folder which will increase the size of their bundle :s and there is no way we can disable the assets from being copied.

Therefore, as I explained in the rationale I mentioned above, they can use whatever icon set they may like and they should add the necessary entry to the angular.json file to copy those to their outputh path, in this case they are also free to decide what the outputh folder will be.